File: frontend/src/pages/backlog/model/issue.ts
interface Epic {
  id: number;
  name: string;
  key: string;
}

export interface Issue {
  id: number;
  name: string;
  key: string;
  bizPoint: number;
  issueImportance: string;
  issueStatus: string;
  component: {
    id: number;
    name: string;
  };
  assignee: {
    id: number;
    nickname: string;
    profileImageUrl: string;
  };
  epic?: Epic;
}

File: backend/core/core-domain/src/main/java/com/ssafy/taskit/domain/Issue.java
package com.ssafy.taskit.domain;

import com.ssafy.taskit.domain.support.DefaultDateTime;

public record Issue(
    Long id,
    String name,
    String content,
    String key,
    Long bizPoint,
    Importance issueImportance,
    IssueStatus issueStatus,
    Long componentId,
    Long assigneeId,
    Long epicId,
    Long sprintId,
    Double position,
    DefaultDateTime defaultDateTime) {

  public boolean isReadyToStart() {
    return name != null
        && bizPoint != null
        && issueImportance != null
        && componentId != null
        && assigneeId != null
        && epicId != null;
  }
}
